cell_based/src/population/forces/FarhadifarForce.cpp

```diff
@@ -196,18 +196,36 @@ double FarhadifarForce<DIM>::GetLineTensionParameter(Node<DIM>* pNodeA, Node<DIM
     return line_tension_parameter_in_calculation;
 }
 
+template<unsigned DIM>
+double FarhadifarForce<DIM>::GetLineTensionParameter(unsigned elem_index, Node<DIM>* pNodeA, Node<DIM>* pNodeB, VertexBasedCellPopulation<DIM>& rVertexCellPopulation)
+{
+  return GetLineTensionParameter(pNodeA, pNodeB, rVertexCellPopulation);
+}
+
 template<unsigned DIM>
 double FarhadifarForce<DIM>::GetAreaElasticityParameter()
 {
     return mAreaElasticityParameter;
 }
 
+template<unsigned DIM>
+double FarhadifarForce<DIM>::GetAreaElasticityParameter(unsigned elem_index, VertexBasedCellPopulation<DIM>& rVertexCellPopulation)
```

Comment:
This overload does not seem to be necessary: the new parameters are not used in any way.